changelogs/fragments/6-routeros_facts-multiple-entries-per-iface.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
```

I think this would be what can go into the new `breaking_changes` category:
```suggestion
breaking_changes:
```